astropy/nddata/nduncertainty.py

```diff
@@ -378,3 +396,6 @@ def propagate_divide(self, other_nddata, result_data):
              result_data)
 
         return result_uncertainty
+
+uncertainty_type_dict = {item.uncertainty_type:item
+                         for item in NDUncertainty.__subclasses__()}
```

Comment:
To be clear, a disadvantage of this approach is that if a module containing an `NDUncertainty` subclass is imported later than this module, it won't be automatically added to this.